src/Tizen.Applications.Common/Tizen.Applications/SuspendedStateEventArgs.cs

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// The class for the argument of the SuspendedState EventHandler
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class SuspendedStateEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes SuspendedStateEventArgs class
+        /// </summary>
+        /// <param name="state">The information of the SuspendedState</param>
+        /// <since_tizen> 6 </since_tizen>
+        public SuspendedStateEventArgs(SuspendedState state)
+        {
+            SuspendedState = state;
+        }
+
+        /// <summary>
+        /// The property to get the information of the SuspendedState
```

Comment:
Suggestion.

        /// Gets the suspended state of applications.
or 
        /// Gets or sets the suspended state of applications. <= In case of correct but I think it is not suitable because set method is private.